dbt/adapters/default/impl.py

```diff
@@ -273,17 +304,51 @@ def expand_target_column_types(cls, profile,
                 cls.alter_column_type(profile, to_schema, to_table,
                                       column_name, new_type, model_name)
 
+    ###
+    # RELATIONS
+    ###
+    @classmethod
+    def list_relations(cls, profile, model_name=None):
+        raise dbt.exceptions.NotImplementedException(
```

Comment:
I think we'll want to implement this, right? It's called below in `get_relations`, and I think we can do it as long as we have a schema. We'll probably need to add an argument for that